addons/units/items/base_helpers.hpp

```diff
@@ -67,6 +67,12 @@ class CLASS(Unit_Combat_LS_Base): B_Soldier_base_F {
     hiddenSelectionsMaterials[] = {QPATHTOF(data\uniform_combat.rvmat)};
 };
 
+// Combat - Long Sleeves - Atacs
+
+class CLASS(Unit_Combat_LS_Atacs_Base): CLASS(Unit_Combat_LS_Base) {
```

Comment:
```suggestion
class CLASS(Unit_Combat_LS_ATACS_Base): CLASS(Unit_Combat_LS_Base) {
```
And everywhere else.